### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by idungstanley**
## Description

- Database setup already
- Ran a query to add an extra column to the owner's table
-  Insert some statements to add data to the database

## Project Requirement

- Ran this queries SELECT COUNT(*) FROM visits where animal_id = 4;
   - Before using explain analyse 
   analyze-1
   - After using explain analyse 
   - image
- Ran this queries SELECT * FROM visits where vet_id = 2;
   - Before the explain analyse 
analyze-2
   - After using the explain analyse 
image
- Ran this queries SELECT * FROM owners where email = 'owner_18327@mail.com';
   - Before using the explain analyse 
analyze-3
   - After using the expalin analyse 
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -37,3 +37,5 @@ SELECT owners.id, full_name FROM animals JOIN owners ON animals.owner_id = owner
 EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
 
 EXPLAIN ANALYZE SELECT  vet_id FROM visits WHERE vet_id = 2;
+
+EXPLAIN ANALYZE SELECT full_name, email FROM owners WHERE email = 'owner_18327@mail.com';
```
Comment (Kingstalux) [inline queries.sql:41]: - [x] Good job so far 👍 but the goal of this project is to improve the performance of your queries. As a result, the queries should have a lower execution time after the indexes have been applied but I can see that all your queries are run using `Seq scan`. If the indexes are applied correctly, you should see an `index scan` on the screenshot. Kindly work on this issue 💯

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @idungstanley and @Fatima-hub333 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
175083599-879bf673-fb93-406f-939c-de7e586bf6d9


## Highlights 👏 
✅ Good commit history 👍 
✅ Good PR description 🔥 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (idungstanley) [reply queries.sql:41]: ## Project Requirement

- Ran this queries SELECT COUNT(*) FROM visits where animal_id = 4;
   - Before using explain analyse 
   analyze-1
   - After using explain analyse 
   - image
- Ran this queries SELECT * FROM visits where vet_id = 2;
   - Before the explain analyse 
analyze-2
   - After using the explain analyse 
image
- Ran this queries SELECT * FROM owners where email = 'owner_18327@mail.com';
   - Before using the explain analyse 
analyze-3
   - After using the expalin analyse 
image

**Review by ShahierNashaat — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**